CatNote.BLL/Models/Enums/TastStatusModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CatNote.BLL.Models.Enums
+{
+    public enum TastStatusModel
```

Comment:
TaskStatusModel

Reply:
With ReSharper it would be easier to notice such mistakes in advance